### PR TITLE
sgt-back-end | Angel Leung

### PR DESCRIPTION
**Pull request by angelsleung**
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

File: sgt-back-end/index.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  }
+  const sql = `
+    insert into "grades" ("name", "course", "score")
+         values ($1, $2, $3)
+      returning *
+  `;
+  const values = [name, course, score];
+  db.query(sql, values)
+    .then(result => {
+      res.status(200).json(result.rows[0]);
```
Comment (thebearingedge) [inline sgt-back-end/index.js]: Successful creation should receive a `201` (Created) response.

File: sgt-back-end/index.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    .then(result => {
+      const grade = result.rows[0];
+      if (!grade) {
+        res.status(404).json({
+          error: `Unable to find grade with 'gradeId' ${gradeId}`
+        });
+      } else {
+        res.status(200).json({
+          deleted: grade
+        });
```
Comment (thebearingedge) [inline sgt-back-end/index.js]: Here you can `res.sendStatus(204)`.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Excellent, just a couple microscopic changes.

**Comment by angelsleung** [comment]
image
image

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**